recipes/pyzstd/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pyzstd" %}
+{% set version = "0.14.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 43df9c969b912874f78edf2d29b78710547bff89c0e61c63aaa8af5ab824588b
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install --install-option="--dynamic-link-zstd" -vv .
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - pip
+    # ZDICT_finalizeDictionary() function is
+    # available since zstd v1.4.5.
+    - zstd >=1.4.5
```

Omit the constraint so that it uses the global pinning. This is already at `1.4.9` so well above your constraint.
```suggestion
    - zstd
```